index.html

```diff
@@ -1917,28 +1917,29 @@
                 If <var>record</var>.recordType is <code>undefined</code>, then:
                 <ol> <!-- guess type and mediaType from data -->
                   <li>
-                    If <var>record</var>.data is instance of
-                    <code>ArrayBuffer</code>, then
-                    set <var>record</var>.mediaType to
-                    <code>"application/octet-stream"</code> and
-                    <var>record</var>.recordType to <code>"opaque"</code>.
+                    If the type of <var>record</var>.data is an
```

Comment:
The spec is using IDL types for interface definition, so, it validates those against IDL types. ToType(x) or Type(x) are ecma bindings.